### PR TITLE
Adds Stroopwafels

### PR DESCRIPTION
**Pull request by PacifistDalek**
## What Does This PR Do
This is all @farie82 and @dearmochi 's fault.

Adds the classic dutch treat stroopwafels to the station as a coin good for the getmore chocolate chip vending machines. 

## Why It's Good For The Game
More delicious noms are good. Coin goods are fun, and having a slightly more healthy option in the Getmore vendors can be useful if the kitchen explodes.

## Images of changes
image
image

## Changelog
:cl:
add: Adds stroopwafels
/:cl:

### CODE REVIEW

**Review by SteelSlayer — APPROVED** [review]

File: code/modules/food_and_drinks/food/foods/junkfood.dm [hunk 1]
```diff
@@ -84,6 +84,14 @@
 	list_reagents = list("nutriment" = 2, "sugar" = 4)
 	tastes = list("bread" = 1)
 
+/obj/item/reagent_containers/food/snacks/stroopwafel
+	name = "stroopwafel"
+	desc = "Straight from the snack factories of Space Den Haag comes a classic dutch treat!"
+	icon_state = "stroopwafel"
+	filling_color = "#EDB878"
+	junkiness = 20
```
Comment (hal9000PR) [inline code/modules/food_and_drinks/food/foods/junkfood.dm:92]: Bit of a nitpick. If these are supposed to be premium and rare I dont think it really makes sense for them to have a high `junkiness`

**Review by Kyep — APPROVED** [review]

File: code/modules/food_and_drinks/food/foods/junkfood.dm (hunk 1 above)
Comment (PacifistDalek) [reply code/modules/food_and_drinks/food/foods/junkfood.dm:92]: A good point!

**Review by dearmochi — APPROVED** [review]
YUM.

**Comment by hal9000PR** [comment]
Is it a cookie or a pastry?

**Comment by farie82** [comment]
> Is it a cookie or a pastry?

More a cookie than pastry. Since it is *just* two wafers combined with syrup in between. Which are made from cookies

**Comment by Spacemanspark** [comment]
Sounds like a sandwich or hotdog to me

**Comment by lectronyx** [comment]
Good idea, made even better by the fact that "Stroopwafel" is really fun to say out loud

**Comment by PacifistDalek** [comment]
> 
> 
> Sounds like a sandwich or hotdog to me

It's smaller than that, it's like the diameter of the top of a mug of your preferred warm beverage. It's also sweet :)

**Comment by Spacemanspark** [comment]
>It's smaller than that, it's like the diameter of the top of a mug of your preferred warm beverage. It's also sweet :)

To that, I raise you an American classic:



Sandwiches do not discriminate between sweetness nor size.

C h e c k m a t e

**Comment by farie82** [comment]
> > It's smaller than that, it's like the diameter of the top of a mug of your preferred warm beverage. It's also sweet :)
> 
> To that, I raise you an American classic:
> 
> sandwich
> 
> Sandwiches do not discriminate between sweetness nor size.
> 
> C h e c k m a t e

You Americans are truly horrifying at times

**Review by variableundefined — APPROVED** [review]

**Pull request merged**